### PR TITLE
EVG-5859 scrub error of git.get_project

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: command/git.go [hunk 1]
```diff
@@ -573,3 +580,15 @@ func (c *gitFetchProject) applyPatch(ctx context.Context, logger client.LoggerPr
...
 	return nil
 }
+
+type noopWriteCloser struct {
+	w *bytes.Buffer
+}
+
+func (c noopWriteCloser) Write(data []byte) (n int, err error) {
+	return c.w.Write(data)
+}
```
Comment (tychoish) [inline command/git.go]: I think if you make the *bytes.Buffer anonymous, then you don't have to implement this. 

also avoid declaring n and err in the signature...

**Comment by john-m-liu** [comment]
evergreeen merge

File: command/git.go (hunk 1 above)
Comment (john-m-liu) [reply command/git.go]: fixed

**Comment by john-m-liu** [comment]
evergreen merge

**Pull request merged**